schemas.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel, validator
+
+#inputi type validation
```

Comment:
Small typo. Not sure you need this comment but won't object if you want to keep it.